sql/seed-tables.sql

```diff
@@ -0,0 +1,9 @@
+--Inserts data into User's table
+
+INSERT INTO  users (firstname, lastname, email, password)
+VALUES
+('James','Bond','james.bond@gmail.com','b6b7fb4cad4bc020f76e16889a8e9065cb708d0f8c304a8a3db609b644da9536'),
```

Comment:
Can you add more users?

Reply:
Future users will be added into postgres database